Sources/StreamVideoSwiftUI/CallingViews/MicrophoneChecker.swift

```diff
@@ -22,20 +20,22 @@ public class MicrophoneChecker: ObservableObject {
     
     private let audioFilename: URL = {
         let documentPath = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask)[0]
-        let audioFilename = documentPath.appendingPathComponent("recording.m4a")
+        let audioFilename = documentPath.appendingPathComponent("recording.aac")
```

Comment:
not related to your PR but + i have no idea how this works but: isn't `recording.aac` going to grow forever on the device? is there a way to use a tmp file and perhaps rotate it?

Reply:
AVAudioRecorder overwrites the file everytime we call `audioRecorder.record()` (to be exact the file is being overriden when we call `[prepareToRecord](https://developer.apple.com/documentation/avfaudio/avaudiorecorder/1389435-preparetorecord)` which is happening implicitly every time we call `record`)

Reply:
don't we have this running for the whole duration of the call?

Reply:
Not yet. We are implementing it as part of #102